pkg/auth/auth_token_handler.go

```diff
@@ -141,13 +145,74 @@ func (a *AuthTokenHandlerImpl) refreshCCloudSSOToken(client *ccloudv1.Client, re
 	return res.GetToken(), refreshToken, err
 }
 
-func (a *AuthTokenHandlerImpl) GetConfluentToken(mdsClient *mdsv1.APIClient, credentials *Credentials) (string, error) {
+func (a *AuthTokenHandlerImpl) GetConfluentToken(mdsClient *mdsv1.APIClient, credentials *Credentials, noBrowser bool) (string, string, error) {
 	ctx := utils.GetContext()
```

Comment:
Didn't know we had this function... maybe we should try to deprecate/remove it? The standard approach to getting the context is from the (un)authenticated command struct

Reply:
Oh, meaning from the stdlib "context" package?

Reply:
Huh, I'm looking at the implementation of this. Why do we need to print DNS/TLS information here?

Reply:
Hmm. I'm not sure. Maybe we can replace both calls to this w/ background context and remove it like you suggested at the start.